legal-api/tests/unit/models/test_filing.py

```diff
@@ -274,3 +276,59 @@ def test_get_filing_by_payment_token(session):
 
     assert rv
     assert rv.payment_token == payment_token
+
+
+def test_get_filings_by_status(session):
+    """Assert that a filing can be retrieved by status."""
+    uow = versioning_manager.unit_of_work(session)
+    transaction = uow.create_transaction(session)
+    business = factory_business('CP1234567')
+    payment_token = '1000'
+    filing = Filing()
+    filing.business_id = business.id
+    filing.filing_json = AR_FILING
+    filing.payment_token = payment_token
+    filing.transaction_id = transaction.id
+    filing.payment_completion_date = datetime.datetime.utcnow()
+    filing.save()
+
+    rv = Filing.get_filings_by_status(business.id, [Filing.Status.COMPLETED.value])
+
+    assert rv
+    assert rv[0].status == Filing.Status.COMPLETED.value
+
+
+# testdata pattern is ({str: environment}, {expected return value})
+TEST_FILING_GO_LIVE_DATE = [
```

Comment:
Because dates in JS (and possibly Python) are fickle, how about testing "on the day of" and "one day before" to include/exclude filings?
In other words, if Go Live Date is 2019-08-12, test with a filing date of 2019-08-12 (should be included) and 2019-08-11 (should not be included)?
I just think +10 and -10 won't catch edge errors :)